.circleci/config.yml

```diff
@@ -811,9 +811,6 @@ commands:
       - run:
           name: append rust-fil-sector-builder checksum
           command: git rev-parse @:./proofs/rust-fil-sector-builder >> /tmp/rust-submodules-checksums.txt
```

Comment:
@sidke - How will CI behavior change once you’ve removed bls-signatures from the rust-submodules-checksum file?

Reply:
i mistakenly assumed some things about how submodules worked in the ci cache key generations, i'm pushing up a change that will put it back in